apps/sv/frontend/src/hooks/useVoteRequestResultByCid.tsx

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) 2024 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { ContractId } from '@daml/types';
+import { useVoteRequest } from './useVoteRequest';
+import {
+  DsoRules_CloseVoteRequestResult,
+  VoteRequest,
+} from '@daml.js/splice-dso-governance/lib/Splice/DsoRules';
+import { useVotesHooks } from '@lfdecentralizedtrust/splice-common-frontend';
+import { Contract } from '@lfdecentralizedtrust/splice-common-frontend-utils';
+
+const QUERY_LIMIT = 50;
```

Comment:
What happens if there are more?

Reply:
Known issue: https://github.com/hyperledger-labs/splice/issues/1372

Reply:
Fix PR'd

Reply:
Fix for current UI PR'd https://github.com/hyperledger-labs/splice/pull/1388